### DISCUSSION TITLE
setting graded_spike = True for slayer.neuron.cuba.Neuron causes to hang the program

### DISCUSSION THREAD

**Discussion by gwgknudayanga** [lava-nc/lava-dl#349] [Q&A]
Hi ,

When i set the graded_spike = True for the LIF neuron, the program is hanging.

neuron_params_2 = {
                'threshold'     : 0.0,
                'graded_spike'  : True,
                'current_decay' : 0.25,
                'voltage_decay' : 0.03,
                'tau_grad'      : 0.03,
                'scale_grad'    : 3,
                'requires_grad' : False,
            }
        self.head = torch.nn.ModuleList([
            slayer.synapse.Conv(16,10,kernel_size=1, stride=1, padding=0,pre_hook_fx = quantizer),
            slayer.neuron.cuba.Neuron(**neuron_params_2),
        ])
